lenses/yaml.aug

```diff
@@ -28,8 +28,18 @@ View: indent
 *)
 let indent = del /[ \t]+/ "  "
 
-let mval = [ label "@mval" . Util.del_str "|-" . eol
-           . [ label "@line" . indent . store Rx.space_in . eol ]+ ]
+let mline = [ label "@line" . indent . store Rx.space_in . eol ]+
+
+let m_literal_clip = [ label "@mlitclip" . Util.del_str "|" . eol . mline ]
+let m_literal_strip = [ label "@mval" . Util.del_str "|-" . eol . mline ]
+let m_literal_keep = [ label "@mlitkeep" . Util.del_str "|+" . eol . mline ]
+
+let m_fold_clip = [ label "@mfoldclip" . Util.del_str ">" . eol . mline ]
+let m_fold_strip = [ label "@mfoldstrip" . Util.del_str ">-" . eol . mline ]
+let m_fold_keep = [ label "@mfoldkeep" . Util.del_str ">+" . eol . mline ]
```

Comment:
Instead og having all these different labels, how about doing something like
```
let mval = [ label "@mval" . store /[|>][+-]?/ . eol . mline ]
```

Reply:
I'm very green at augeas, so I'll be happily led by others.

I was trying to avoid storing syntactical symbols in the augeas result.  labels looked like they could be a nicer way of showing the meaning for end-users.
Is there a way to have two labels?  `[|>]` and `[+-]` say two different things.